jetbrains-core/src/software/aws/toolkits/jetbrains/services/ecs/EcsExplorerNodes.kt

```diff
@@ -0,0 +1,86 @@
+// Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.services.ecs
+
+import com.intellij.openapi.project.Project
+import icons.AwsIcons
+import software.amazon.awssdk.services.ecs.EcsClient
+import software.amazon.awssdk.services.ecs.model.Service
+import software.aws.toolkits.jetbrains.core.AwsResourceCache
+import software.aws.toolkits.jetbrains.core.explorer.AwsExplorerService
+import software.aws.toolkits.jetbrains.core.explorer.nodes.AwsExplorerEmptyNode
+import software.aws.toolkits.jetbrains.core.explorer.nodes.AwsExplorerNode
+import software.aws.toolkits.jetbrains.core.explorer.nodes.AwsExplorerResourceNode
+import software.aws.toolkits.jetbrains.core.explorer.nodes.AwsExplorerServiceRootNode
+import software.aws.toolkits.jetbrains.core.explorer.nodes.ResourceParentNode
+import software.aws.toolkits.jetbrains.services.ecs.resources.EcsResources
+import software.aws.toolkits.resources.message
+
+class EcsParentNode(project: Project) : AwsExplorerServiceRootNode(project, AwsExplorerService.ECS) {
+    override fun getChildrenInternal(): List<AwsExplorerNode<*>> = listOf(
+        EcsClusterParentNode(nodeProject),
+        EcsTaskDefinitionFamiliesParentNode (nodeProject)
+    )
+
+    override fun isAlwaysShowPlus(): Boolean = true
```

Comment:
isn't this the exact definition as the parent `AwsExplorerServiceRootNode` ?

Reply:
opps, ya. Why you no warn IntelliJ?!